elixir/apps/web/assets/js/hooks.js

```diff
@@ -9,7 +9,10 @@ Hooks.Copy = {
     this.el.addEventListener("click", (ev) => {
       ev.preventDefault();
 
-      let text = ev.currentTarget.querySelector("[data-copy]").innerHTML.trim();
+      let inner_html = ev.currentTarget.querySelector("[data-copy]").innerHTML.trim();
```

Comment:
`.innerHTML` returns a percent-encoded string, so we copy-pasted code that wasn't able to run if it included any `&` or `<` characters.